Src/PRuntimes/PJavaRuntime/pom.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>p.runtime</groupId>
+    <artifactId>PJavaRuntime</artifactId>
+    <version>1.0</version>
```

Comment:
Hehe, its done through the tag using the release workflow

Reply:
https://github.com/p-org/P/blob/master/.github/workflows/release.yml

Reply:
It would be great to add the maven release as a part of this release workflow, what do you think?

Reply:
> It would be great to add the maven release as a part of this release workflow, what do you think?

This would be good!  One problem though is that we don't have a place for the Jar to live - there's no good equivalent to NuGet for Java Jar storage (that I can find; everything seems to depend on paying some company money).  We could at minimum generate the Jar but just not upload it anywhere for the time being (though that seems like a bit of a waste of cycles), what do you think?

Reply:
Yeah lets just skip it.

Reply:
Yeah, we can do it manually at first and then hopefully have a story for putting that jar somewhere down the road.